baseball_game.py

```diff
@@ -282,45 +283,60 @@ def is_no(one_more_input):
 def main():
     print("Play Baseball")
 
+    end_game = False
     while True:
         user_input = "999"
     
         # make random number
         random_number = str(get_not_duplicated_three_digit_number())
         print("Random Number is : ", random_number)
 
-        # get input
         while True:
```

Comment:
unless 3 strike,
1. do not change random_number (was problem - solved : add "[293] while True:", modified optional codes)
    - this problem causes 'unittest assertion'
2. get new user_input